[DOCU-3184] New topic: MinK/Mesh getting started guide (#5457)
Kong/docs.konghq.com: app/konnect/getting-started/configure-mesh.md

1. Click **Create Zone**. 
1. Enter "eu" in the **Name** field for the new zone, and then click **Create Zone & generate token**. 
    {:.note}
    > **Note:** The zone name must consist of lower case alphanumeric characters, `.`, or `-`. It must also start and end with an alphanumeric character.
1. Ensure both [**Zone Ingress**](/mesh/latest/production/cp-deployment/zone-ingress/) and [**Zone Egress**](/mesh/latest/production/cp-deployment/zoneegress/) are enabled.
1. Follow the instructions that display to set up Helm and a secret token. 
    Once {{site.konnect_short_name}} finds the newly created mesh, it will display it. 
1. Click **Next**.

You now have a very basic {{site.mesh_product_name}} service mesh added to {{site.konnect_short_name}}. This service mesh can't do anything at the moment until we add services and additional configurations to it.

## Configure `kumactl` to connect to your global control plane

`kumactl` is a CLI tool that you can use to access {{site.mesh_product_name}}. It can do the following:

> * Perform read-only operations on {{site.mesh_product_name}} resources on Kubernetes. 

Reviewer: @johnharris85 , to confirm, drop line 52?

> * Perform read-only operations on {{site.mesh_product_name}} resources on Kubernetes. 

Reviewer: NM, refers to line 51, so have commented out.

* Read and create resources in {{site.mesh_product_name}} in Universal mode.

You connect `kumactl` to the global control plane in {{site.konnect_short_name}} so that you can run commands against the control plane.

1. In {{site.konnect_short_name}}, select your user icon to open the context menu and click **Personal access tokens**, then click **Generate token**.
1. Enter "control-plane" in the **Name** field and click **Generate**.
1. Copy the personal access token that displays.
1. From the left navigation menu in {{site.konnect_short_name}}, open [MESH MANAGER ICON HERE] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager) and select the `example-cp` control plane.
1. Copy your control plane ID.

> 1. Run the following to ?:

Author: ```suggestion
1. Run the following to connect your global control plane to {{site.product_mesh_name}}:
```

    ```sh

>     kumactl config control-planes add --address https://us.api.konghq.com/v0/mesh/control-planes/MGCP_ID/api --name mink-dev --headers 'authorization=Bearer KPAT'

Reviewer 2: ```suggestion
    kumactl config control-planes add --address https://us.api.konghq.com/v0/mesh/control-planes/MGCP_ID/api --name mesh-konnect --headers 'authorization=Bearer KPAT'
```

    ```
    * **MGCP_ID:** Replace this with the control plane ID.
    * **KPAT:** Replace this with the personal access token you created for the control plane.

You can now run commands against your global control plane using `kumactl`. You can see the [`kumactl` command reference](/mesh/latest/generated/cmd/kumactl/kumactl/) for more information about the commands you can use.

## Add services to your service mesh using the Kubernetes demo app

> Now that you've added a global control plane and a zone to your service mesh in {{site.konnect_short_name}}, you can add services to your mesh. You can use the {{site.mesh_product_name}} Kubernetes demo app to set up two services so you can see how {{site.mesh_product_name}} can be used to control services, monitor traffic, and track resource status.

Reviewer 2: ```suggestion
Now that you've added a global control plane and a zone to your service mesh in {{site.konnect_short_name}}, you can add services to your mesh. You can use the {{site.mesh_product_name}} Kubernetes demo app to set up four services so you can see how {{site.mesh_product_name}} can be used to control services, monitor traffic, and track resource status.
```

> The demo application consists of two services:

Reviewer 2: ```suggestion
The demo application consists of four services:
```

> * `demo-app`: A web application that lets you increment a numeric counter

Reviewer 2: ```suggestion
* `frontend`: A web application that lets you browse an online clothing store
```

> * `redis`: A data store for the counter

Reviewer 2: ```suggestion
* `backend`: A nodejs API for querying and filtering clothing items
* `postgres`: A database for storing clothing item reviews
* `redis`: A data store for the clothing item star ratings
```

To add the services to your mesh using the demo app, run the following:

```sh
kubectl apply -f https://raw.githubusercontent.com/kumahq/kuma-demo/master/kubernetes/kuma-demo-aio.yaml
```

You can see the services the Kubernetes demo app added by navigating to **Mesh Manager** in the sidebar of {{site.konnect_short_name}}, selecting the `example-cp` and clicking **Meshes** in the sidebar. You can view the services associated with that mesh by clicking **Default** and the **Services** tab.

> After you add a service to your service mesh, the {{site.konnect_short_name}} will be populated with metrics from your service mesh. For more information about the Kubernetes demo app, see [Explore {{site.mesh_product_name}} with the Kubernetes demo app](/mesh/latest/quickstart/kubernetes/).

Reviewer 2: ```suggestion
For more information about the Kubernetes demo app, see [Explore {{site.mesh_product_name}} with the Kubernetes demo app](/mesh/latest/quickstart/kubernetes/).
```

## Conclusion

> By following the instructions in this guide, you've created a {{site.mesh_product_name}} global control plane, added a zone to it, configured `kumactl` to connect to your global control plane, and added services to the control plane. 

Reviewer 2: ```suggestion
By following the instructions in this guide, you've created a {{site.mesh_product_name}} global control plane, added a zone to it, configured `kumactl` to connect to your global control plane, and added services to the mesh. 
```

Now that you've completed this tutorial, you can continue to configure your service mesh in {{site.konnect_short_name}} by following some of these guides:

> * [Zone Ingress](/mesh/latest/production/cp-deployment/zone-ingress/) - Set up zone ingress in {{site.mesh_product_name}}.

Reviewer 2: This will already be configured, we can delete this line.

Reviewer: Done

> * [Zone Egress](/mesh/latest/production/cp-deployment/zoneegress/) - Set up zone egress in {{site.mesh_product_name}}.

Reviewer 2: This will already be configured, we can delete this line.

Reviewer: Done

* [Mutual TLS](/mesh/latest/policies/mutual-tls/) - Configure mTLS with {{site.mesh_product_name}}. 
* [Observability](/mesh/latest/explore/observability/) - Find out how to configure observability with {{site.mesh_product_name}}.
* [Traffic Log](/mesh/latest/policies/traffic-log/) - Learn how to configure logging with {{site.mesh_product_name}}.

## Next steps

> Now that you've configured a global control plane, you can continue by [importing {{site.base_gateway}} entities into {{site.konnect_short_name}}](/getting-started/import/).

Reviewer 2: I think this is just copied over from the Kong guide, don't think it makes sense here. I'm in favor of moving the remaining 3 items from above into this `Next Steps` section.

![mesh global control plane](/assets/images/diagrams/gslides/kuma_multizone.svg)

> _**Figure 1:** {{site.mesh_product_name}} can support multiple zones (like a Kubernetes cluster, VPC, data center, etc.) together in the same distributed deployment. Then, you can create multiple isolated virtual meshes with the same control plane in order to support every team and application in the organization._

For the purposes of this guide, you'll use {{site.mesh_product_name}} as your service mesh to create a global control plane. You will use Kubernetes to install various service mesh components.

## Prerequisites

* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
* [Download the latest version of {{site.mesh_product_name}}](/mesh/latest/production/install-kumactl/)

## Create a global control plane in {{site.konnect_short_name}}
 

> 1. From the left navigation menu in {{site.konnect_short_name}}, open [MESH MANAGER ICON HERE] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager).

Reviewer: ```suggestion
1. From the left navigation menu in {{site.konnect_short_name}}, open [/assets/images/icons/konnect/icn-mesh-manager.svg] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager).
```

1. Click **New Control Plane**.
1. Enter "example-cp" in the **Name** field.
1. Click **Save**.

You now have a {{site.mesh_product_name}} global control plane. This control plane can't do anything at the moment because we need to connect a zone to it next. 

## Create a zone in the global control plane

After creating the global control plane, you must add a zone to that control plane. Adding a zone allows you to manage services added to that zone and send and receive configuration changes to the zone. 

> 1. In [MESH MANAGER ICON HERE] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager), click the `example-cp` control plane you just created, and then click **Zones** in the sidebar.

Reviewer: ```suggestion
1. In [/assets/images/icons/konnect/icn-mesh-manager.svg] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager), click the `example-cp` control plane you just created, and then click **Zones** in the sidebar.
```

1. Follow the instructions that display to set up Helm and a secret token. 
    Once {{site.konnect_short_name}} finds the newly created zone, it will display it. 
1. Click **Next**.

You now have a very basic {{site.mesh_product_name}} service mesh added to {{site.konnect_short_name}}. This service mesh can't do anything at the moment until we add services and additional configurations to it.

## Configure `kumactl` to connect to your global control plane

`kumactl` is a CLI tool that you can use to access {{site.mesh_product_name}}. It can do the following:

<!--* Perform read-only operations on {{site.mesh_product_name}} resources on Kubernetes. -->
* Create, read, update, and delete resources in {{site.mesh_product_name}} in Universal/{{site.konnect_short_name}} mode.

You connect `kumactl` to the global control plane in {{site.konnect_short_name}} so that you can run commands against the control plane.

1. In {{site.konnect_short_name}}, select your user icon to open the context menu and click **Personal access tokens**, then click **Generate token**.
1. Enter "control-plane" in the **Name** field and click **Generate**.
1. Copy the personal access token that displays.

> 1. From the left navigation menu in {{site.konnect_short_name}}, open [MESH MANAGER ICON HERE] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager) and select the `example-cp` control plane.

Reviewer: ```suggestion
1. From the left navigation menu in {{site.konnect_short_name}}, open [/assets/images/icons/konnect/icn-mesh-manager.svg] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager) and select the `example-cp` control plane.
```
